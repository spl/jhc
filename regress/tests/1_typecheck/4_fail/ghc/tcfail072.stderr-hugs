ERROR "tcfail072.hs":23 - Cannot justify constraints in explicitly typed binding
*** Expression    : g
*** Type          : (Ord a, Ord b) => AB a b -> Bool
*** Given context : (Ord a, Ord b)
*** Constraints   : (Ord c, Ord d)
