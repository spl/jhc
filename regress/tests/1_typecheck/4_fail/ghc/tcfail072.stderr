
tcfail072.hs:23:13:
    Ambiguous type variable `p0' in the constraint:
      (Ord p0) arising from a use of `g'
    Probable fix: add a type signature that fixes these type variable(s)
    In the expression: g A
    In an equation for `g': g (B _ _) = g A

tcfail072.hs:23:13:
    Ambiguous type variable `q0' in the constraint:
      (Ord q0) arising from a use of `g'
    Probable fix: add a type signature that fixes these type variable(s)
    In the expression: g A
    In an equation for `g': g (B _ _) = g A
